hack/verify-generated-bindata.sh

```diff
@@ -4,7 +4,7 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 function cleanup() {
     return_code=$?
     rm -rf "${TMP_GENERATED_BOOTSTRAP_DIR}"
-    os::test::junit::generate_report
+    # os::test::junit::generate_report
```

Comment:
uncomment, right?

Reply:
`os::test::junit::generate_report` does not exist.  We need to delete all of this and move to the lib-go `make` stuff anyway.

Reply:
oh i see, nm, we need to update CI/update to library-go shell scripts, etc.